ush/config_defaults.yaml

```diff
@@ -416,6 +416,14 @@ platform:
   COMINgfs: ""
   COMINgefs: ""
   COMINairnow: "/path/to/real/time/airnow/data"
+  #
+  #-----------------------------------------------------------------------
+  #
+  # observation file templates used in process_bufr
+  #
+  #-----------------------------------------------------------------------
+  #
+  OBSPATH_TEMPLATE: '${OBSPATH}/${YYYYMMDDHH}.${obs_source}'
```

Comment:
Should this one go in the process_bufr section? Is it needed by other tasks?

Reply:
Moved.